Commands/notify.py

```diff
@@ -160,9 +160,10 @@ async def save_config(self, ctx):
         await save_configs(ctx)
         await ctx.send("Configurations saved")
 
-    @commands.command(name="add_list", brief="**admin-only** \n\u2003 Add a new list",usage='<list name>', help="*admin-only* \u2003 Add a new list. \n You will be asked what emoji to use for this list. React to the question of the bot with an emoji that is not yet used for another list. \n\n <list name> \u2003 The name of the list to add.")
+    @commands.command(name="add_list", brief="**admin-only** \n\u2003 Add a new list", usage='<list name>', help="*admin-only* \u2003 Add a new list. \n You will be asked what emoji to use for this list. React to the question of the bot with an emoji that is not yet used for another list. \n\n <list name> \u2003 The name of the list to add.")
     async def add_list(self, ctx, list_name):
-        if not ctx.message.author.guild_permissions.administrator:
+        guild_data = await get_guild_data(ctx.message.guild.id)
+        if not (ctx.message.author.guild_permissions.administrator or ctx.author.id in guild_data.bot_admins):
```

Comment:
This seems like something we would use often, maybe we should extract both lines into a funtion
`check_admin`

Reply:
in which file would we put check_admin? could use it in my feature as well. GuildData I would suggest since it is not something that only applies to notify.py then. pass it `ctx.message` as an argument

Reply:
GuildData seems logical, since it contains the data needed in the check

Reply:
You would still need to get the guild data because it is also used later in the code, so you would replace 1 line of code with a function and 1 line of different code. Not sure if that is worth it?

Reply:
it avoids copy pasting that code, though, so the admin check logic can be changed if needed without doing a search and replace later on to change all the places where it's been put.